### PR TITLE
Home

### PR DESCRIPTION
**Pull request by rajaranjanmr**
adding home page and category section

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
```
Comment (sri-rishi) [inline src/App.css:28]: Suggestion: use variable color naming

File: src/Pages/CategoryPage.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <div class="category cat-three">
+                <div class="category-content">Programming knowledge</div>
+                <a class="button-category" href="rule.html">play now</a>
+
+            </div>
+            <div class="category cat-four">
+                <div class="category-content cat5">Politics hist</div>
+                <a class="button-category" href="rule.html">play now</a>
+
+            </div> */}
```
Comment (sri-rishi) [inline src/Pages/CategoryPage.js]: Suggestion: if commented code is not usable later , then remove them, ignore if useable

**Review by sri-rishi — APPROVED** [review]
code looks good, just some suggestion which I think can be done, else feel free to merge

File: src/Pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+function Home(){
+    const {state,dispatch} = useQuiz();
+    useEffect(()=>{
+        async function getCategories(){
+            try{
+                const response = await axios({
+                    method: "GET",
+                    url: "https://opentdb.com/api_category.php",
+                  });
+                  console.log("its response",response.data.trivia_categories)
```
Comment (nikhiljugale007) [inline src/Pages/Home.js]: remove console.log

**Review by nikhiljugale007 — COMMENTED** [review]
added few suggestion, rest code is good.

**Pull request merged**